.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,58 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Before filing a bug please check the following:**
+- There's no existing/similar bug report.
+- This bug report is about a single actionable bug.
+- This bug is about the Pro Git book, version 2, English language.
+- This bug is about the book as found on the [website](https://www.git-scm.com/book/en/v2) or the pdf.
+- If you found a issue in the pdf, you've checked if the problem is also present in the Pro Git book on the [website](https://www.git-scm.com/book/en/v2).
+- This bug is **not** about a translation, if so please file a bug with the translation project. You can find a table of translation projects here: [progit2/TRANSLATING.md](https://github.com/progit/progit2/blob/master/TRANSLATING.md)
+- This bug is **not** about the .epub or .mobi files.
```

Comment:
You can remove this line. The epub and mobi files are generated for the git-scm.com site with the same tools as the pdf.

Reply:
Line removed.